mne/io/eeglab/eeglab.py

```diff
@@ -226,6 +233,12 @@ class RawEEGLAB(_BaseRaw):
         Names or indices of channels that should be designated
         EOG channels. If 'auto', the channel names containing
         ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+    event_id : dict
+        By default, events are read from the input file by dropping every
+        non-integer part of events containing integers, and completely
+        dropping any events without integer parts. If non-integer events
+        should be read, this should be a dict mapping from their names to
+        integers, e.g. dict(fmri_scan_onset=199, recording_start=255).
```

Comment:
> I've thought about this, but consider this solution superior. 1. this is how BV does it. 
ok
> 2. most users use mostly int codes, and many readers already (silently) drop the non-integer parts of an event id (for example, the BV and EGI readers do this). 
silent stuff is always scary
> 3. you want to predictably map the same triggers to the same event_ids across subjects, ruling out many ways of automatically generating mappings.
okay, I like this point. But you can sort the unique event keys and then assign them trigger codes? Kind of what we do in `plot_events` ...